### DISCUSSION TITLE
Why Gray Alpha HEIF colorspace and chromaspace are RGB in libvips？

### DISCUSSION THREAD

**Discussion by jiaoyiren00** [libvips/libvips#2667] [Q&A]
Why GrayA PNG file Transcoding To HEIF picture，the colorspace and chroma is set to heif_colorspace_RGB and heif_chroma_interleaved_RGBA？

           _error = heif_image_create( heif->page_width, heif->page_height, 
		heif_colorspace_RGB, 
		vips_image_hasalpha( heif->image ) ?
			heif_chroma_interleaved_RGBA : 
			heif_chroma_interleaved_RGB,
		&heif->img );_

However，in libheif example file，I found the colorspace and chroma is set to heif_colorspace_monochrome and 
heif_chroma_monochrome？

           _err = heif_image_create((int) width, (int) height,
                            heif_colorspace_monochrome,
                            heif_chroma_monochrome,
                            &image);_

**Comment by jcupitt** [comment]
Sure, let's make an enhancement issue.
